charts/consul/templates/connect-inject-deployment.yaml

```
@@ -96,13 +96,15 @@ spec:
                   name: {{ .Values.connectInject.aclInjectToken.secretName }}
                   key: {{ .Values.connectInject.aclInjectToken.secretKey }}
             {{- end }}
+            {{- if not .Values.externalServers.enabled }}
             - name: CONSUL_HTTP_ADDR
               {{- if .Values.global.tls.enabled }}
               value: https://{{ template "consul.fullname" . }}-server.{{ .Release.Namespace }}.svc:8501
               {{- else }}
               value: http://{{ template "consul.fullname" . }}-server.{{ .Release.Namespace }}.svc:8500
             {{- end }}
```

super-nit: but I think the alignment is off on this one? maybe its my screen.
```suggestion
              {{- end }}
```

---

oh sorry; missed this one! I'll fix it in my next PR